dev/custom_split.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
```

Comment:
it'll be great if you could have an brief usage/function description in comment blocks in terms of what does this script do.

Reply:
Sure, it was used as the basis for the tool, i was afraid of actually commenting in the tool itself, but can add comments here

Reply:
Ok, added to script and added doc description to tool